docs/git.md

```diff
@@ -0,0 +1,112 @@
+# Git events
+
+Git events are related to commits in git repositories.
+
+| Field | Type | Description |
+|-------|------|-------------|
+| attributes | `Object` ([attributes](#attributes)) | Context attributes |
+| data | `Object` ([data](#data)) | Event-specific payload|
+
+
+## Attributes
+
+Metadata providing contextual information about the event, based on the CloudEvents specification.
+Shared by all git event types, except for the `linked_event` field, only found in [file actions](#commit-file-actions) event types.
```

Comment:
The link to file actions doesn't work, is it `#file-objects`?

It also happens in the next references to the same category

Reply:
This particular link should point to `#commit-file-actions`, but I've fixed the links to `#file-objects` in the commit and merge sections that weren't working.